CONTRIBUTING.md

```diff
@@ -54,6 +54,11 @@ Try to keep lines of code no longer than 160 characters wide. This isn't a stric
 
 Try to keep your coding style in line with the existing code. It might not exactly match your preferred style but it's better to keep things consistent. StyleCop compliance is enforced through analysis on each build and the treatment of warnings as errors. Any StyleCop settings changes or suppressions must be clearly justified.
 
+## String Formatting
+
+Unless there is a compelling reason not to, for example when serializing data to be parsed later, use the current culture when formatting strings. When string
+formatting methods have an overload that implicitly uses the current culture, opt to use it instead of specifying the culture explicitly.

```

Comment:
> Why should explicit string.Format calls be persecuted?

Because the people who designed the code analysis rules vowed to make developers' lives miserable. I can't think of any other reason :P